react/website/Datamatics/Datamatics/Resolvers/HeaderItemResolver.cs

```diff
@@ -0,0 +1,65 @@
+﻿using Datamatics.Models;
+using Sitecore.Data.Fields;
+using Sitecore.Data.Items;
+using Sitecore.LayoutService.Configuration;
+using Sitecore.LayoutService.ItemRendering.ContentsResolvers;
+using Sitecore.Mvc.Presentation;
+using System;
+using System.Collections.Generic;
+using System.Collections.Specialized;
+
+namespace Datamatics.Resolvers
+{
+    public class HeaderItemResolver : IRenderingContentsResolver
+    {
+        public bool IncludeServerUrlInMediaUrls { get; set; }
+        public bool UseContextItem { get; set; }
+        public string ItemSelectorQuery { get; set; }
+        public NameValueCollection Parameters { get; set; }
+        public object ResolveContents(Rendering rendering, IRenderingConfiguration renderingConfig)
+        {
+            Item contextItem = Sitecore.Context.Database.GetItem("/sitecore/content/sitecorejsspoc/Content/PageComponents/Header Component");
```

Comment:
passing the component as part of datasource and fetching it form datasource can be more effective solution, then hard coding the path, as the path and names might change..